bcrs-ui/src/assets/styles/base.scss

```diff
@@ -0,0 +1,321 @@
+@import "theme.scss";
+
+@font-face {
+  font-family: 'BCSans';
+  font-style: normal;
+  font-weight: 400;
+  src: url('../fonts/BCSans/BCSans-Regular.woff2') format('woff2'), /* Optimized for very modern browsers */
+       url('../fonts/BCSans/BCSans-Regular.woff') format('woff'); /* Modern Browsers */
+}
+@font-face {
+  font-family: 'BCSans';
+  font-style: italic;
+  font-weight: 400;
+  src: url('../fonts/BCSans/BCSans-Italic.woff2') format('woff2'), /* Optimized for very modern browsers */
+       url('../fonts/BCSans/BCSans-Italic.woff') format('woff'); /* Modern Browsers */
+}
+@font-face {
+  font-family: 'BCSans';
+  font-style: normal;
+  font-weight: 700; 
+  src: url('../fonts/BCSans/BCSans-Bold.woff2') format('woff2'), /* Modern Browsers */
+       url('../fonts/BCSans/BCSans-Bold.woff') format('woff'); /* Optimized for very modern browsers */
+}
+@font-face {
+  font-family: 'BCSans';
+  font-style: italic;
+  font-weight: 700; 
+  src: url('../fonts/BCSans/BCSans-BoldItalic.woff2') format('woff2'), /* Modern Browsers */
+       url('../fonts/BCSans/BCSans-BoldItalic.woff') format('woff'); /* Optimized for very modern browsers */
+}
+
+html,
+body {
+  min-height: 100vh;
+}
+
+html {
+  // default font size (14px)
+  font-size: 0.875rem;
+}
+
+@media (min-width: 960px) {
+  html {
+    // default font size (16px)
+    font-size: 1rem;
+  }
+}
+
+body {
+  font-size: 1rem;
+}
+
+h1, h2, h3, h4 {
+  color: $gray9;
+}
+
+h1, h2, h3 {
+  letter-spacing: -0.04rem;
+  font-weight: 700;
+}
+
+h1 {
+  line-height: 2.25rem;
+  font-size: 2rem;
+}
+
+p {
+  font-weight: 400;
+}
+
+.v-application {
+  font-family: 'BCSans', Verdana, Arial, sans-serif;  // BC Government Font
+}
+
+.theme--light.v-application {
+  color: $gray7;
+  background-color: $gray1;
+}
+
+.form-input {
+  display: flex;
+  border: 1px solid $gray5;
+  width: 10rem;
+
+  .prefix {
+    flex: 0 0 auto;
+    width: 2.5rem;
+    padding: 0.5rem;
+    background-color: $gray1;
+    border-right: 1px solid $gray5;
+    text-align: center;
+  }
+
+  input,
+  select {
+    padding-left: 0.5rem;
+    height: 36px;
+  }
+
+  input {
+    flex: 1 1 auto;
+    width: 100%;
+  }
+
+  select {
+    flex: 1 1 auto;
+    -webkit-appearance: menulist;
+  }
+}
+// Payment Table
```

Comment:
I think we aren't using these 3 classes anywhere (not this project and not coops-ui). Please verify and delete this obsolete CSS.

Reply:
okie doke, will do!